tools/release/sync-gutenberg-packages.js

```diff
@@ -120,8 +120,8 @@ function getMismatchedNonWordPressDependencies() {
 	;
 
 	// Ensure that all the conflicts can be resolved with the same version
-	const unresolvableConflicts = Object.entries( groupBy( versionConflicts, ( [name] ) => name ) )
```

Comment:
The destructuring here was an actual syntax error -- `versionConflicts` is an [array of objects](https://github.com/WordPress/wordpress-develop/pull/4701/files#diff-fba85087e3ce27a12e329f6c6cf0e63ee1c7421d45dbc3cfe1dc73385afc0338R110-R119) (with keys `name`, `required`, `actual`), not an array of arrays.

Reply:
> was an actual syntax error

type error :)

syntax was fine, data types were wrong.